### ISSUE TITLE
[bug] Auto rebuild/restart isn't work in macOS

### ISSUE THREAD

**Issue by BillGoldenWater** [tauri-apps/tauri#4640]
### Describe the bug

When using Intellij Idea to edit source code, the `yarn tauri dev`(and `cargo tauri dev`) will output this:
```
    Blocking waiting for file lock on package cache
    Blocking waiting for file lock on package cache
    Blocking waiting for file lock on package cache
   Compiling test v0.1.0 (/Users/golden_water/IdeaProjects/Test/src-tauri)
✨  Done in 17.18s.
```
and just exit.
if using other thing to edit source code like vscode or just `echo  >>main.rs`, it will correctly rebuild/restart.
if using tauri-cli@1.0.0 rather than higher version, it will correctly rebuild/restart.

### Reproduction

1. in macOS 12.4 Apple Silicon
2. create a tauri app
3. run it using `yarn tauri dev`
4. edit source code in Intellij Idea

### Expected behavior

expect it will correctly rebuild/restart

### Platform and versions

```shell
Environment
  › OS: Mac OS 12.4.0 X64
  › Node.js: 16.14.0
  › npm: 8.5.1
  › pnpm: Not installed!
  › yarn: 1.22.18
  › rustup: 1.24.3
  › rustc: 1.62.0
  › cargo: 1.62.0
  › Rust toolchain: stable-aarch64-apple-darwin 

Packages
  › @tauri-apps/cli [NPM]: 1.0.3
  › @tauri-apps/api [NPM]: 1.0.2
  › tauri [RUST]: 1.0.3,
  › tauri-build [RUST]: 1.0.3,
  › tao [RUST]: 0.12.1,
  › wry [RUST]: 0.19.0,

App
  › framework: React

App directory structure
  ├─ dist
  ├─ node_modules
  ├─ src-tauri
  ├─ .git
  ├─ .idea
  └─ src
```

### Additional context

In another env(also macOS Apple Silicon) it will correctly rebuild/restart, but still exit after that, cause a empty window open(the dev server is stopped because `yarn tauri dev` is exited)

**Comment by lucasfernog** [comment]
I believe this was fixed in #4684, you can check installing `cargo tauri` from git: `cargo install tauri-cli --git https://github.com/tauri-apps/tauri` and running `cargo tauri dev` again. Let me know if it's still an issue.

**Comment by BillGoldenWater** [comment]
fixed! thanks!
